src/app/views/collections/create/CreateNewCollection.jsx

```diff
@@ -146,6 +146,15 @@ const CreateNewCollection = props => {
         return new Date(`${newCollection.publishDate.value} ${newCollection.publishTime.value}`).toISOString();
     };
 
+    const makeTeams = () => {
+        return newCollection.teams.map(team => {
```

Comment:
Maybe a comment explaining what has changed in the passed in props or what is expected by the api with the PUT